docs/docs/micromatch-wrapper.md

```diff
@@ -0,0 +1,52 @@
+#### Overview
+[Micromatch](https://github.com/micromatch/micromatch) support various matching features, such as Glob pattern, Advanced globbing.

```

Comment:
[Micromatch](https://github.com/micromatch/micromatch) supports various matching features, such as glob patterns and advanced globbing.

Reply:
yes, ok